apps/ex_nvr/priv/repo/migrations/20231207093851_device_override_on_full_disk.exs

```diff
@@ -0,0 +1,31 @@
+defmodule ExNVR.Repo.Migrations.DeviceOverrideOnFullDisk do
+  use Ecto.Migration
+
+  def up do
```

Comment:
Delete this file, there's no need to run this migration, the fields will be created the next time we update the device. For getting the values,  it'll return the default values.